gef-ui/templates/partials/unissued-change-about-facility.njk

```diff
@@ -0,0 +1,273 @@
+              },
+              errorMessage: errors and errors.fieldErrors.shouldCoverStartOnSubmission and {
+                text: errors.fieldErrors.shouldCoverStartOnSubmission.text,
+                attributes: {
+                  'data-cy': 'should-cover-start-on-submission-error'
+                }
+              },
+              items: [
+                {
+                  value: 'true',
+                  text: "Yes",
+                  checked: shouldCoverStartOnSubmission === 'true',
+                  attributes: {
+                    'data-cy': "should-cover-start-on-submission-yes"
+                  }
+                },
+                {
+                  value: 'false',
+                  text: "No",
+                  checked: shouldCoverStartOnSubmission === 'false',
+                  conditional: {
+                    html: coverStartHtml
+                  },
+                  attributes: {
+                    'data-cy': "should-cover-start-on-submission-no"
+                  }
+                }
+              ]
+            }) }}
+            {{ govukDateInput({
+              id: "coverEndDate",
+              namePrefix: "cover-end-date",
+              fieldset: {
+                legend: {
+                  text: "When do you want the UKEF cover to end?",
+                  classes: "govuk-fieldset__legend--s"
+                }
+              },
+              hint: {
+                text: "For example, 12 03 2021."
+              },
+              errorMessage: errors and errors.fieldErrors.coverEndDate and {
+                text: errors.fieldErrors.coverEndDate.text,
+                attributes: {
+                  'data-cy': 'cover-end-date-error'
+                }
+              },
+              items: [
+                {
+                  label: "Day",
+                  classes: errors.fieldErrors.coverEndDate and "govuk-input--error govuk-input--width-2" or "govuk-input--width-2",
+                  name: "day",
+                  value: coverEndDateDay,
+                  attributes: {
+                    'data-cy': "cover-end-date-day"
+                  }
+                },
+                {
+                  label: "Month",
+                  classes: errors.fieldErrors.coverEndDate and "govuk-input--error govuk-input--width-2" or "govuk-input--width-2",
+                  name: "month",
+                  value: coverEndDateMonth,
+                  attributes: {
+                    'data-cy': "cover-end-date-month"
+                  }
+                },
+                {
+                  label: "Year",
+                  classes: errors.fieldErrors.coverEndDate and "govuk-input--error govuk-input--width-4" or "govuk-input--width-4",
+                  name: "year",
+                  value: coverEndDateYear,
+                  attributes: {
+                    'data-cy': "cover-end-date-year"
+                  }
+                }
+              ]
+            }) }}
```

Comment:
Could we either:
- move these inputs into their own marcros/components and import into this file

Or use existing input fields used on other facility pages? Presuming they're the same?

Reply:
todo: dtfs2-5228